localstack/services/sqs/sqs_listener.py

```diff
@@ -5,7 +5,7 @@
 from six.moves.urllib.parse import urlencode
 from requests.models import Request, Response
 from localstack import config
-from localstack.config import HOSTNAME_EXTERNAL
+from localstack.config import HOSTNAME_EXTERNAL, SQS_PORT_EXTERNAL
```

Comment:
We cannot import `SQS_PORT_EXTERNAL` here as it hasn't been exposed as a variable in `config.py`.

There are two options here:
* Remove the import, and retrieve the value in line 63 via `int(os.environ.get('SQS_PORT_EXTERNAL') or 0)`
* Expose the `<SERVICE>_PORT_EXTERNAL` variables in `config.py`, similar to [`PORT_<SERVICE>`](https://github.com/localstack/localstack/blob/master/localstack/config.py#L147-L148)